schema/gene.js

```diff
@@ -67,14 +68,18 @@ const Gene = {
       type: new GraphQLNonNull(GraphQLString),
     },
   },
-  resolve: (root, { id }, request, { fieldASTs }) => {
+  resolve: (root, { id }, request, other) => {
```

Comment:
Oh, that’s probably when used through the `node` interface? I think it’s because I call the `resolve` function of the resulting type in [ObjectIdentification](https://github.com/artsy/metaphysics/blob/da3181da9cf6213786d731d902b3d038f802782d/schema/object_identification.js#L100-L101) directly 😁